.github/workflows/build.yml

```diff
@@ -64,10 +64,10 @@ jobs:
           java-version: 11
           distribution: temurin
           cache: 'gradle'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
```

Comment:
Important: `@v3` exists, and is running Node 20, AND it's [RC1](https://github.com/gradle/actions/releases)!

Reply:
Good to know!

Reply:
Meanwhile [stable](https://github.com/gradle/actions/releases/tag/v3.0.0). 🥳